### PR TITLE
Issue #8540, add classifier for jetty-10.0.x

### PR DESCRIPTION
**Pull request by MoonLord-LM**
fix：https://github.com/eclipse/jetty.project/issues/8540

### CODE REVIEW

**Comment by MoonLord-LM** [comment]
@olamy

**Review by olamy — APPROVED** [review]

File: jetty-bom/pom.xml [hunk 1]
```diff
@@ -210,6 +210,7 @@
       <dependency>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>infinispan-remote</artifactId>
+        <classifier>config</classifier>
```
Comment (joakime) [inline jetty-bom/pom.xml:213]: What?

Comment (MoonLord-LM) [reply jetty-bom/pom.xml:213]: https://repo1.maven.org/maven2/org/eclipse/jetty/infinispan-remote/10.0.11/

[https://repo1.maven.org/maven2/org/eclipse/jetty/infinispan-remote/10.0.11/infinispan-remote-10.0.11-<b>config</b>.jar](https://repo1.maven.org/maven2/org/eclipse/jetty/infinispan-remote/10.0.11/infinispan-remote-10.0.11-config.jar)

Comment (joakime) [reply jetty-bom/pom.xml:213]: Depending on the `config` classifier (only meaningful to `jetty-home`) in a bom is just super odd, and feels wrong.
Why not just have `<type>pom</type>` instead?

Comment (MoonLord-LM) [reply jetty-bom/pom.xml:213]: Yes, I agree, `<type>pom</type>` is better.

Comment (joakime) [reply jetty-bom/pom.xml:213]: @olamy can you think of any maven technical reason for not just using `<type>pom</type>` instead?

Comment (olamy) [reply jetty-bom/pom.xml:213]: the bom is mostly made (e.g only useful) for embedded users.
So TBH I don't even understand why we have `<artifactId>infinispan-remote</artifactId>` in the bom. This module contains no code but only configuration files.
So except the `config` jar which contains the jetty configuration module files it's a totally useless artifact :) 
Either we remove it from the bom or we "expose" the only useless artifact which is the `config` but frankly I even don't understand why :) 
wdy? Remove the entry from the bom?

Comment (joakime) [reply jetty-bom/pom.xml:213]: > Remove the entry from the bom?

I'd vote +1 for this approach.

Comment (olamy) [reply jetty-bom/pom.xml:213]: FTR https://github.com/eclipse/jetty.project/pull/8554

**Pull request merged**